### ISSUE TITLE
Attach to Debug Logs from Emulator

### ISSUE THREAD

**Issue by StuartFerguson** [Malinskiy/action-android#16]
Is there a way with this action to show the debug logs written out by the Android Emulator similar to this option https://docs.microsoft.com/en-us/xamarin/android/deploy-test/debugging/android-debug-log?tabs=windows available in Visual Studio.

**Comment by Malinskiy** [comment]
@StuartFerguson would you be okay having a log file attached to the action execution? I can read the logcat synchronously and output that to stdout of the action but I doubt that will be useful because there might be other output to stdout at the same time, to be honest.

What do you think?

**Comment by StuartFerguson** [comment]
So that would be available as an artifact of the action?  If yes that would be fine with me to be honest.

**Comment by another-dave** [comment]
I'd really like this too. Have step starting OK & the emulator boots but then hangs on me for 20min and it's a bit of a black box to try and fix.

For me, either in line or artifact at the end would be great, if I could capture the artifact on cancel/failure (or always :) ) as that's when I would want to debug

**Comment by Malinskiy** [comment]
This is solved in 0.0.6. Use the snippet below to capture logs on failed execution:
```yaml
- name: Save logcat output
  uses: actions/upload-artifact@master
  if: failure()
  with:
    name: logcat
    path: artifacts/logcat.log
```

The actual log is saved at `artifacts/logcat.log`

Feel free to close the issue if this addresses your use-case @StuartFerguson
